kodilla-good-pattern/src/main/java/com/kodilla/good/patterns/challanges/allegro/OrderProcessor.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.good.patterns.challanges.allegro;
+
+public class OrderProcessor {
+    private InformationService informationService;
+    private OrderService orderService;
+    private OrderRepository orderRepository;
+
+    public OrderProcessor(InformationService informationService, OrderService orderService, OrderRepository orderRepository) {
+        this.informationService = informationService;
+        this.orderService = orderService;
+        this.orderRepository = orderRepository;
+    }
+
+    public OrderDto process(final OrderRequest orderRequest) {
+        boolean isRented = orderService.order(orderRequest.getUser(), orderRequest.getSubject(),
```

Comment:
skoro przekazujesz wszystkie rzeczy z OrderRequest to po prostu przekaż orderRequest, a wyciąganie zmiennych zrób w kolejnej metodzie